src/index.html

```diff
@@ -2,21 +2,107 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"
     />
```

Comment:
Use 2 spaces for indentation in your file (not a tab character) and remember about correct indentation between parent and child elements. The <meta> tags should be indented with 2 spaces relative to the <head> tag.